doc/users/project/code_of_conduct.rst

```diff
@@ -0,0 +1,147 @@
+.. code_of_conduct
+
+====================================
+Contributor Covenant Code of Conduct
```

Comment:
Orthogonal to this PR, but came up because you needed a modified entry in the TOC:

Is this the "Contributor Covenant" code of conduct, or is it " adapted from the `Contributor Covenant` "? 

This headline is missing punctuation or something because "Contributor Covenant Code of Conduct" doesn't mean anything in plain English, and only makes sense if you know what "Contributor Covenant" is, which we don't explain until the bottom.  

On-topic: I don't see what is wrong with moving this, but maybe some people don't want to click through?